src/Employer.Web/Configuration/IoC.cs

```diff
@@ -12,14 +12,29 @@ public static class IoC
     {
         public static void AddIoC(this IServiceCollection services, IConfiguration configuration)
         {
+            //Recruit.Storage.Client IoC
+            services.AddRecruitStorageClient(configuration);
+
+            //Configuration
             services.Configure<ExternalLinksConfiguration>(configuration.GetSection("ExternalLinks"));
+            
             services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>(); // Used by NLog to log out traceidentifier value.
 
             services.AddTransient<IGetAssociatedEmployerAccountsService, GetAssociatedEmployerAccountsService>();
 
-            //Mediatr
-            services.AddMediatR(typeof(CreateVacancyCommandHandler).Assembly);
-            services.AddTransient<IMessaging, MediatrMessaging>();
+            //Orchestrators
```

Comment:
I'm going to be shot down 🔫 as usual but `#region` #justsayin 😋 

Because it is static we can't even move it to a partial class 😞

Reply:
At some point we may need to do this.  Its not too long at the mo so I'm leaving it for now.  Maybe we could group similar setups into their own .Addxxx extensions. E.g. services.AddOrchestrators()